src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/JsonEncodingStreamWrapper.cs

```diff
@@ -267,15 +267,39 @@ public override void Write(byte[] buffer, int offset, int count)
             }
         }
 
+        public override void Write(ReadOnlySpan<byte> spanBuffer)
```

Comment:
This shouldn't be duplicating the code from `Write(byte[] buffer, int offset, int count)`. The Write(byte[], ...)` method should just delegate to the span-based one.